crates/ruff_minimizer/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "ruff_minimizer"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
```

Comment:
What's the motivation for this to be its own crate vs making it part of the `cargo dev` crate? We could also start splitting `cargo dev` into multiple sub crates, see https://github.com/rome/tools/tree/main/xtask

Reply:
having less dependencies / faster compilation, but i'm happy to move it anywhere